aerleon/lib/nokiasrl.py

```diff
@@ -52,9 +52,12 @@
     "ACLEntry",
     {"sequence-id": int, "action": Action, "match": Match},
 )
-Entries = TypedDict("Entries", {"entry": List[ACLEntry], "description": str})
+Entries = TypedDict(
+    "Entries", {"entry": List[ACLEntry], "description": str, "name": str, "_annotate": str}
```

Comment:
You added name but did you want to add the tests for them?

Reply:
It's covered by the updated .ref files, still not sure how the TypedDict would be tested/enforced